src/sentry/api/endpoints/organization_events_trends_v2.py

```diff
@@ -129,15 +129,9 @@ def get_timeseries(top_events, _, rollup, zerofill_results):
             ]
             queries = [generate_top_transaction_query(t_e) for t_e in split_top_events]
 
-            timeseries_columns = cast(list[str], selected_columns[:])
+            timeseries_columns = selected_columns[:]
             timeseries_columns.append(trend_function)
 
-            # When all projects or my projects options selected,
-            # keep only projects that top events belong to to reduce query cardinality
-            used_project_ids = list({event["project"] for event in data})
-
-            request.GET.projectSlugs = used_project_ids  # type: ignore[attr-defined]
-
             # Get new params with pruned projects
```

Comment:
this appears to be completely bogus -- and the type checker was helpfully pointing it out and was ignored :(

Reply:
I agree that this is poorly written but I think deleting this has the effect of scanning additional rows in the following clickhouse query.

Do you mind reverting this part and prune `pruned_params` below to contain just the projects in `data`? I can also take this part from you and push a fix today if you want.

Reply:
the `projectSlugs` token does not appear in the codebase at all -- I don't think this is doing what is intended -- and it's certainly not demonstrated in a test

could you point me at why you think this code is doing something?

Reply:
I believe it's done here https://github.com/getsentry/sentry/blob/master/src/sentry/api/bases/organization.py#L339

Reply:
I fixed this in #67701, feel free to either merge that one or pull the changes in this PR